test/data/testinit.js

```diff
@@ -1,11 +1,17 @@
 /* eslint no-multi-str: "off" */
 
-// baseURL is intentionally set to "data/" instead of "".
-// This is not just for convenience (since most files are in data/)
-// but also to ensure that urls without prefix fail.
-// Otherwise it's easy to write tests that pass on test/index.html
-// but fail in Karma runner (where the baseURL is different).
-var baseURL = "data/",
+var FILEPATH = "/test/data/testinit.js",
+	activeScript = [].slice.call( document.getElementsByTagName( "script" ), -1 )[ 0 ],
+	parentUrl = activeScript && activeScript.src ?
+		activeScript.src.replace( /[?#].*/, "" ) + FILEPATH.replace( /[^/]+/g, ".." ) + "/" :
+		"../",
+
+	// baseURL is intentionally set to "data/" instead of "".
```

Comment:
format: style mismatch:
```javascript
	// baseURL is intentionally set to "data/" instead of "".
```


Replace `<-tab>` with `<+tab>` at column 1.
Confidence: 0.983
Triggered rule #26:
```
    left_1_diff_line > 0.500
    right_0_length > 1.500
    left_3_reserved in |;|
    left_0_internal_type in |stringliteral|
    left_2_reserved in |;|
    left_1_label in |<newline>|
    left_0_reserved in |;|
    right_0_roles in |function˙literal|
    left_0_label in |<space>|
```